### PR TITLE
update a README file

### PR DESCRIPTION
**Pull request by anutech2001**
comment #1

### CODE REVIEW

**Review by kidd2001 — CHANGES_REQUESTED** [review]
I thing u add too many line.

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
-# example
\ No newline at end of file
+# example
+
+change something.
+
```
Comment (anutech2001) [inline README.md:4]: Is this correct.

**Review by kidd2001 — APPROVED** [review]
ok

File: README.md (hunk 1 above)
Comment (kidd2001) [reply README.md:4]: yes
```suggestion
abc
```

**Review by kidd2001 — APPROVED** [review]

**Comment by kidd2001** [comment]
thank you for your contribution.

**Pull request merged**